### PR TITLE
re-arranged files and updated some

### PR DESCRIPTION
**Pull request by cyberGHostJs**
i made some changes to the file arrangements and i also added the react router dependency and updated the reUsable button, added 404 page not found

### CODE REVIEW

**Comment by cyberGHostJs** [comment]
re-arranged the file arrangements, added 404 page, updated the re-reusable button and added detailed comment

**Comment by cyberGHostJs** [comment]
re-arranged the file arrangements, added 404 page, updated the re-reusable button and added detailed comment

**Comment by cyberGHostJs** [comment]
re-arranged the file arrangements, added 404 page, updated the re-reusable button and added detailed comment

File: src/components/ReusableButton.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import {} from "@mui/material";
 import React from "react";
 
-function ReusableButton() {
+function ReusableButton({ btn_txt }) {
+  //this makes the button reusable such that when ever u wanna make use of the button you only have to enter the btn text as a string
```
Comment (Ienergyup-innovationlab) [inline src/components/ReusableButton.js:5]: yeah i understand this

**Comment by Ienergyup-innovationlab** [comment]
All the changes have been reviewed and are valid. thanks for the good work

**Pull request merged**